app/models/company.rb

```diff
@@ -19,33 +19,33 @@ class Company < ApplicationRecord
 
   private
 
+  # TODO: Fix this as may break with new naming setup @Dan
+  # TODO: Do we call this? If not then remove
+
```

Comment:
can delete standardize_name and also COMPANY_NAME_FILTER on line 14 above.

Reply:
agreed